petools/core/pose_predictor_interface.py

```diff
@@ -59,10 +63,13 @@ def predict(self, image: np.ndarray):
         pass
 
     @staticmethod
-    def pack_data(humans, end_time, **kwargs):
+    def pack_data(humans: List[Human], end_time, **kwargs):
         data = {
             PosePredictorInterface.HUMANS: [
-                (single_human.id, single_human.to_dict(prepend_p=True), single_human.to_dict_from3d(prepend_p=True))
+                (
+                    single_human.id, single_human.to_dict(prepend_p=True), single_human.to_dict_from3d(prepend_p=True),
+                    single_human.pose_name, single_human.pose_class_conf
+                )
```

Comment:
Maybe we should overload __repr__ method to make this part independent of the Human class?